addons/binding/pom.xml

```diff
@@ -1,59 +1,60 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not sure why it shows this whole file being changed. Maybe revert the change and only add the single new line with your binding.

Please also add the binding here: https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml